routes/api.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+const db = require("../db/models");
+const { asyncHandler } = require("./utils");
+
+const createReview = db.Review
```

Comment:
Once you destructure `Review` Then you don't need this variable, so please delete this line.

Reply:
Fixed!